<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphEditWidget</class>
 <widget class="QDialog" name="GraphEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>921</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="drawNameButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>laba 6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>50</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colorSelect">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backgroundButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>change back</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>137</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="brushButton">
      <property name="text">
       <string>brush</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="brushSizeSpinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>137</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="eraserButton">
      <property name="text">
       <string>eraser</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="eraserSizeSpinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="brushStyleComboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
